pkgs/applications/networking/instant-messengers/pidgin-plugins/pidgin-osd/default.nix

```diff
@@ -2,21 +2,19 @@
 , autoreconfHook } :
 
 stdenv.mkDerivation rec {
-  name = "pidgin-osd-0.1.0";
+  name = "pidgin-osd-0.2.0";
   src = fetchurl {
-    url = https://github.com/mbroemme/pidgin-osd/archive/pidgin-osd-0.1.0.tar.gz;
-    sha256 = "11hqfifhxa9gijbnp9kq85k37hvr36spdd79cj9bkkvw4kyrdp3j";
+    url = https://github.com/edanaher/pidgin-osd/archive/pidgin-osd-0.2.0.tar.gz;
+    sha256 = "1dfb0957wwm5zly4w4g815svhkhvjbj3dgik1lbyac8adh9ks784";
```

Comment:
Could you change the version number the same or change to `-unstable-<year>-<month>-<day>` scheme. Otherwise this would irritate users.